kazel/generator.go

```diff
@@ -17,120 +17,146 @@ limitations under the License.
 package main
 
 import (
-	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"regexp"
 	"sort"
 	"strings"
-)
 
-const (
-	openAPITagName = "openapi-gen"
-	staging        = "staging/src/"
+	bzl "github.com/bazelbuild/buildtools/build"
 )
 
 var (
 	// Generator tags are specified using the format "// +k8s:name=value"
 	genTagRe = regexp.MustCompile(`//\s*\+k8s:([^\s=]+)(?:=(\S+))\s*\n`)
 )
 
-// walkGenerated updates the rule for kubernetes' OpenAPI generated file.
-// This involves reading all go files in the source tree and looking for the
-// "+k8s:openapi-gen" tag. If present, then that package must be supplied to
-// the genrule.
-func (v *Vendorer) walkGenerated() error {
-	if !v.cfg.K8sOpenAPIGen {
-		return nil
+// tag name -> (pkg -> values)
```

Comment:
nit: is this standard golang symantics? Would think something closer to `{tagName: {pkg: [values]}}` would be easier to read.

Reply:
fair enough, my [ML](https://en.wikipedia.org/wiki/ML_(programming_language)) heritage is showing.